app/views/govuk_publishing_components/components/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,20 @@
+<%
+  add_gem_component_stylesheet("secondary-navigation")
+
+  id ||= nil
```

Comment:
This `id` is not exposed in the `.yml` example, and is also untested.

Reply:
Good point, added an id in the `yml` and added a test for it. 
At the same time we noticed the two tests that were there could really be reduced to one since there was some duplication going on.